segmenters/image/FaceNetSegmenter/tests/test_facenetsegmenter.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+import pytest
+
+from PIL import Image
+import numpy as np
+
+from .. import FaceNetSegmenter
+
+
+SEGMENTER_DIR = Path(__file__).parent.parent
+
+
+@pytest.fixture
+def segmenter():
+    return FaceNetSegmenter()
+
+
+@pytest.mark.parametrize('filename', ['one_face.jpg',
+                                      'three_faces.jpg',
+                                      'four_faces.jpg',
+                                      'five_faces.jpg'])
+def test_segment_face(segmenter, filename):
+    image = Image.open(SEGMENTER_DIR / 'imgs' / filename)
+
+    result = segmenter.segment(np.array(image))
+
+    assert isinstance(result, list)
+    assert len(result) == 1
+    assert result[0]['blob'].shape == (3, 160, 160)
```

Comment:
This is duplicated with the test `test_segment_face` at the bottom.

Reply:
Extracted this behavior to a function.